apps/meteor/server/services/video-conference/service.ts

```diff
@@ -1097,8 +1097,15 @@ export class VideoConfService extends ServiceClassInternal implements IVideoConf
 			return;
 		}
 
-		const name = settings.get<string>('VideoConf_Persistent_Chat_Discussion_Name') || i18n.t('Conference Call Chat History');
-		const displayName = `${name} - ${new Date().toISOString().substring(0, 10)}`;
+		const name = settings.get<string>('VideoConf_Persistent_Chat_Discussion_Name') || i18n.t('[date] Video Call Chat');
+		let displayName;
+		const date = new Date().toISOString().substring(0, 10);
+
+		if (name.includes('[date]')) {
```

Comment:
Do you need to check if included? Thought you can just do the `name.replace` which would do nothing if there's no `[date]` inside already

Reply:
just checked to have a case to add the date to the start of the string if no date was provided